projects/dsp-ui/src/lib/viewer/viewer.module.ts

```diff
@@ -25,6 +25,7 @@ import { AddValueComponent } from './operations/add-value/add-value.component';
 import { DisplayEditComponent } from './operations/display-edit/display-edit.component';
 import { StillImageComponent } from './representation/still-image/still-image.component';
 import { BooleanValueComponent } from './values/boolean-value/boolean-value.component';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
```

Comment:
organize imports

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/134/commits/2e7b3456557533e626891e138655faa23a444808